{"additions": 2, "auther_ref": "fix-noexcept-mounan", "auther_sha": "c062f626b8ffeb27229423897944d394e9714746", "author": "davidlt", "body": "throw() dynamic exeption specifier was deprecated in C++11 and finally\r\nremoved in C++17. Use noexcept / noexcept(expression) instead.\r\n\r\nSigned-off-by: David Abdurachmanov <David.Abdurachmanov@cern.ch>", "branch": "CMSSW_9_0_X", "changed_files": 2, "closed_at": "1486564458", "comments": 6, "commits": 1, "created_at": "1486532063", "deletions": 2, "labels": ["analysis-pending", "comparison-available", "orp-pending", "pending-signatures", "tests-approved"], "merge_commit_sha": "6ca228312b0fe53b12e0b82f9a87c131cbbacfdc", "merged_at": "1486564458", "merged_by": "davidlange6", "milestone": "CMSSW_9_0_X", "number": 17455, "release-notes": [], "review_comments": 0, "state": "closed", "title": "Replace throw() dynamic exception specifier with noexcept", "updated_at": "1486564458", "user": "davidlt"}